reark/src/main/java/io/reark/reark/data/stores/cores/ContentProviderStoreCoreBase.java

```diff
@@ -166,12 +182,43 @@ public static Handler createHandler(@NonNull final String name) {
     }
 
     @NonNull
-    private Observable<ContentProviderOperation> createOperation(@NonNull final U item, @NonNull final Uri uri) {
+    private Observable<CoreOperation> createCoreOperation(@NonNull final CoreValue<U> value) {
+        switch (value.type()) {
+            case UPDATE:
```

Comment:
The calls are not the same, one has argument casted to `CoreUpdateValue` and the other to `CoreDeleteValue`